pkg/daemon/update.go

```diff
@@ -1078,9 +1078,9 @@ func (dn *Daemon) applyExtensions(oldConfig, newConfig *mcfgv1.MachineConfig) er
 	}
 
 	// Validate extensions allowlist on RHCOS nodes
-	if err := validateExtensions(newConfig.Spec.Extensions); err != nil && dn.os.IsRHCOS() {
-		return err
-	}
+	// if err := validateExtensions(newConfig.Spec.Extensions); err != nil && dn.os.IsRHCOS() {
```

Comment:
This probably can be dropped - afaik we don't use extensions since 4.8, so it won't affect upgrades.
I'll give that a try later on

Reply:
I believe this would be a no-op on FCOS anyway, due to the `err != nil && dn.os.IsRHCOS()`, no?